review page Tokens (#355)
trilitech/tezos-developer-docs: docs/architecture/tokens.md

  date: 28 November 2023
---

> Any blockchain ecosystem relies on a native token that can be transferred between accounts and in which transaction fees are paid in.

Reviewer: The info about how blockchain ecosystems work is more suited for opentezos. Here we should keep the info to the behavior of Tezos itself for developers.

Author: OK, I re-introduced the original first phrase, and recentered this one on Tezos.

The native token of Tezos is tez (also known as XTZ or represented by the symbol ꜩ).

But other tokens representing some value in digital for can be programmed in a blockchain, for instance using smart contracts.
Tokens fall in two broad categories:

> - Fungible tokens, whose instances are interchangeable and represent the same value,

Reviewer: ```suggestion
- Fungible tokens, which are interchangeable and represent the same value
```
"Whose instances" is wordy; how about just "which?"

Author: thanks, applied

- Non-fungible tokens (NFTs), which are unique digital assets that model the ownership of some digital or real object

Many types of fungible tokens are already implemented in Tezos, including:

- Stablecoins, which are tied to the price of fiat currencies such as USD and EUR
- Wrapped tokens, which represent tokens from another blockchain or another standard; see [Wrapped tokens](#wrapped-tokens)

> Tezos is also reputed as a platform for owning and exchanging various types of NFTs.

Reviewer: ```suggestion
```
Advertising Tezos in this sentence is necessary.

Author: Right, I removed the advertisement phrasing and just said that there are also NFTs, which were also mentioned in the original page.

In most cases, (non-native) tokens are managed by smart contracts.

> They are not stored directly in accounts; instead, smart contracts keep a ledger of how many tokens are hold by different accounts.

Reviewer: ```suggestion
They are not stored directly in accounts; instead, smart contracts keep a ledger of how many tokens each account holds.
```
Making this active to be a little more direct.

Author: Applied, thanks.

- [FA1.2](./tokens/FA1.2) tokens are fungible tokens
- [FA2](./tokens/FA2) tokens can be multiple types of tokens, including fungible and non-fungible tokens, and a single smart contract that follows this standard can create multiple types of tokens

> You can use templates for smart contracts adhering to these standards, instead of writing you own contract from scratch:

Reviewer: ```suggestion
You can use templates for smart contracts adhering to these standards instead of writing your own contract from scratch:
```

Author: applied manually
